config/openshift/v4_10_exp/schema.go

```diff
@@ -36,4 +36,25 @@ type OpenShift struct {
 	Extensions      []string `yaml:"extensions"`
 	FIPS            *bool    `yaml:"fips"`
 	KernelType      *string  `yaml:"kernel_type"`
+	Kdump           Kdump    `yaml:"kdump"`
```

Comment:
The `OpenShift` struct is only for fields [included directly in the `MachineConfigSpec`](https://github.com/openshift/machine-config-operator/blob/master/docs/MachineConfiguration.md#machineconfig-definition), outside of the Ignition config itself.  Also, since kdump configuration is useful for both Fedora CoreOS and RHEL CoreOS, this shouldn't be implemented directly in the `openshift` variant.

Instead, please move kdump support to the `fcos` 1.5.0-experimental spec.  The `openshift` 4.10.0-experimental spec will inherit it from there.  The `Kdump` field can go in the `fcos` `Config` struct.